### PR TITLE
feat: add screenshots and update readme

### PR DESCRIPTION
**Pull request by leite-paulohf**
# Description

This PR contains all necessary context to install and use the SwiftLint correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,105 @@
+<h1 align="center">
+  swiftlint-config-pixelmatters
+</h1>
+<img width="80" src="./img/pixelmatters-logo.jpeg" />
```
Comment (gil0mendes) [inline README.md:4]: You removed the center align paragraph, please add it back.

```suggestion
<p align="center">
  <img width="80" src="./img/pixelmatters-logo.jpeg" />
</p>
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,105 @@
...
+## 🛠️ Usage
+
+After to install, move the file `swiftlint.yml` into the same path of your `Project.xcodeproj`.
+
+### Xcode
+
+Integrate SwiftLint into your Xcode project to get warnings and errors displayed
+in the issue navigator.
+
+![](/Users/pauloleite/Workspace/swiftlint-config-pixelmatters/screenshot.png)
```
Comment (gil0mendes) [inline README.md:68]: To use images you need to add them into a directory (for example, `imgs`) and then use relative paths.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,105 @@
...
+if which swiftlint >/dev/null; then
+ swiftlint --config ".swiftlint.yml"
+else
+ echo "warning: SwiftLint not installed, use: brew install swiftlint"
+fi
+```
+
+After that it needs to look like the screenshot:
+
+![](/Users/pauloleite/Workspace/swiftlint-config-pixelmatters/runscript.png)
```
Comment (gil0mendes) [inline README.md:95]: The same thing as above for this image.

**Review by gil0mendes — CHANGES_REQUESTED** [review]
I left some comments for you to review.

File: README.md (hunk 2 above)
Comment (leite-paulohf) [reply README.md:68]: My bad, I used the Typora Editor to add an image and the editor used my personal path

File: README.md [hunk 4]
```diff
@@ -1,29 +1,105 @@
...
+
+```bash
+if which swiftlint >/dev/null; then
+  swiftlint
+else
+  echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
+fi
+```
+
+### Insert the following as the script for Homebew setup:
```
Comment (FilipeAVS) [inline README.md:83]: Typo in "Homebrew"

**Review by FilipeAVS — CHANGES_REQUESTED** [review]

**Review by FilipeAVS — APPROVED** [review]

**Review by gil0mendes — APPROVED** [review]
Looks good now. Tks for the contribution 🤗

**Pull request merged**